Source/BluemixAppID/internal/AppIDConstants.swift

```diff
@@ -178,6 +178,7 @@ internal class AppIDConstants {
     internal static let USER_IDENTITY_LABEL = "userIdentity"
     // labels
     
+    internal static let AnonymousIdpName = "appid_anon"
     internal static let BMSSecurityErrorDomain = "com.ibm.mobilefirstplatform.clientsdk.swift.bmssecurity"
```

Comment:
What is the BMSSecurityErrorDomain property used for? It is part of MCA SDK, there shouldn't be any mentioning of com.ibm.mobilefirstplatform at all in AppID SDK.